sidebars.js

```diff
@@ -1,6 +1,2 @@
 module.exports = {
-  someSidebar: {
-    Docusaurus: ['doc1', 'doc2', 'doc3'],
-    Features: ['mdx'],
-  },
 };
```

Comment:
This file exports nothing. Do we want to delete the file then?

Reply:
the file has to export something even an empty object. Hopefully we can create multiple sidebar instances and separate them by folder and link to them within the different section plugin. what do you think about that?

Reply:
Oh I see, sounds good